ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -175,16 +175,25 @@
       </else-if>
       <!-- Pour les chapter paper-conference entry-dictionary entry-encyclopedia -->
       <else-if type="chapter paper-conference entry-dictionary entry-encyclopedia" match="any">
-        <choose>
-          <if variable="container-author" match="none">
-            <text macro="editor" prefix=" dans "/>
-            <text variable="container-title" prefix=", " font-style="italic" text-case="capitalize-first"/>
-          </if>
-          <else-if variable="container-author" match="any">
-            <text macro="book_author" prefix=" dans "/>
-            <text variable="container-title" prefix=", " font-style="italic" text-case="capitalize-first"/>
-          </else-if>
-        </choose>
+        <group prefix="dans " delimiter=", ">
```

Comment:
@ahahl, it's better to rely on CSL terms wherever possible. You could e.g. change:

```xml
<group prefix="dans " delimiter=", ">
  ...
</group>
```

to

```xml
<group delimiter=" ">
  <text term="in"/>
  <group delimiter=", ">
     ...
  </group>
```

(you would also have to redefine the "in" term, which is translated as "in" for the "fr-FR" CSL locale file, with:
`<term name="in">dans</term>` )